scripts/publish-deb.sh

```diff
@@ -30,14 +30,17 @@ else
     esac
 
     # only publish wanted jobs
-    if [[ "$CIRCLE_JOB" == "build-artifacts--testnet_postake_medium_curves"  ]]; then
-          cd src/_build
-          echo "Publishing debs:"
-          ls coda-*.deb
-          set -x
-          ${DEBS3} --codename ${CODENAME} --component main coda-*.deb
-          set +x
-    else
-        echo "WARNING: Circle job: ${CIRCLE_JOB} not in publish list"
-    fi
+    case "$CIRCLE_JOB" in
+        build-artifacts--testnet_postake_medium_curves | build-artifacts--testnet_postake_medium_curves_10k_ledger)
```

Comment:
Not now, but we should integrate this part with `test.py` so we only have to maintain one source of truth for artifact builds. Probably the easiest way is to achieve it through a flag to the `publish-deb.sh` script that is only added on CI builds for the build artifacts entries in `test.py`.